src/EnergyPlus/HVACManager.cc

```diff
@@ -3005,6 +3009,30 @@ namespace HVACManager {
         }
     }
 
+    void CheckAirLoopFlowBalance()
+    {
+        // Check for unbalanced airloop
+        if (!DataGlobals::WarmupFlag && AirLoopsSimOnce) {
+            for (int AirLoopNum = 1; AirLoopNum <= NumPrimaryAirSys; ++AirLoopNum) {
+                auto &thisAirLoopFlow(AirLoopFlow(AirLoopNum));
+                if (!thisAirLoopFlow.FlowError) {
+                    Real64 unbalancedExhaustDelta = thisAirLoopFlow.SupFlow - thisAirLoopFlow.OAFlow - thisAirLoopFlow.SysRetFlow;
+                    if (unbalancedExhaustDelta > SmallMassFlow) {
+                        ShowSevereError("CheckAirLoopFlowBalance: AirLoopHVAC " + DataAirSystems::PrimaryAirSystem(AirLoopNum).Name +
+                            " is unbalanced. Supply is > return plus outdoor air.");
+                        ShowContinueErrorTimeStamp("");
+                        ShowContinueError("  Flows [m3/s at standard density]: Supply=" + General::RoundSigDigits(thisAirLoopFlow.SupFlow * DataEnvironment::StdRhoAir, 6) +
```

Comment:
The warning here says "Flows [m3/s at standard density]", and then does SupFlow *StdRhoAir. But SupFlow (and others) is already in kg/s isn't it?

From hh file:
        Real64 SupFlow;               // supply air flow rate (includes LeakFlow) [kg/s]

Reply:
@rraustad Good catch - these are all in kg/s, so need to *divide* by `StdRhoAir` to output m3/s.